README.md

```diff
@@ -268,13 +268,18 @@ As the library is actively developed couple of features are not there yet:
 
 This library uses [semantic versioning](http://semver.org/). For each release, the following needs to be done:
 
-* Create a branch for the release, named like `release-1.1.4`
+* Create a branch for the release, named like `release/1.1.5`
```

Comment:
why `release/1.1.5` instead of `release-1.1.5` ? I mean any reason to use `/` instead of `-`?

Reply:
We had an internal discussion on this a few months back and have decided to start using forward slashes in branch names to delimit categories of branch like this. At the time we updated our internal document [Development flow for Ably repos](https://wiki.ably.io/pages/Development-flow-for-Ably-repos) and I will share that as a snapshot with you via Slack now. The task of formally documenting this in the public domain is [in my backlog](https://github.com/ably/docs/issues/577).